CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns MX on by default
```

Comment:
These two should stay on top.

I'm still thinking if we can have a nicer line for the first one

Reply:
Marco used terms like `That means you can always query distributed tables from any node in a Citus cluster!  `  or `Automatic metadata syncing ` in the draft blog post.

Maybe we can ask @marcocitus's opinion on this first line.

Reply:
maybe: MX -> metadata syncing